src/IO/AbstractIOHandlerHelper.cpp

```diff
@@ -27,10 +27,39 @@
 #include "openPMD/IO/HDF5/HDF5IOHandler.hpp"
 #include "openPMD/IO/HDF5/ParallelHDF5IOHandler.hpp"
 #include "openPMD/IO/JSON/JSONIOHandler.hpp"
+
+#include <fstream>
 #include <nlohmann/json.hpp>
 
 namespace openPMD
 {
+namespace
+{
+    nlohmann::json
+    parseOptions( std::string const & options )
+    {
+        if( options.at( 0 ) == '@' )
+        {
+            std::fstream handle;
+            std::string filename = options;
```

Comment:
Yes, I think implementing a `std::string trim( std::string )` method here is a good idea.